bricksrc/alarm.py

```diff
@@ -252,6 +252,21 @@
                     "Water_Loss_Alarm": {
                         "tags": [TAG.Point, TAG.Loss, TAG.Water, TAG.Alarm],
                     },
+                    "Water_Level_Alarm": {
+                        "tags": [TAG.Water, TAG.Level, TAG.Alarm, TAG.Point],
+                        "subclasses": {
+                            "Collection_Basin_Water_Level_Alarm": {
```

Comment:
Maybe `High` instead of `Collection` because 1) `High` is more straightforward 2) `High` is more commonly used for the same meaning in other classes. If `Collection` contains a special context here, I'm fine with the current this.

Reply:
How is `High` a substitute for `Collection`? `Collection Basin` is the entity containing the water. @corymosiman12 could you help elaborate?

Reply:
Closing this one as I realized taht `Collection_Basin` is a type of Basin.

Reply:
I closed this one as it was my confusion.